File: Csharp/9.cs
public class Solution
{
    public bool IsPalindrome(int x)
    {
        //Convert to string
        string numAsString = x.ToString();
        char[] reversedString = new char[numAsString.Length];
        //convert to array

        for (int i = 1; i <= numAsString.Length; i++) reversedString[numAsString.Length - i] = numAsString[i - 1];
        //check if equal
        if (new string(reversedString) == numAsString) return true;
        else return false;
    }
}

File: JS/9.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
    let xString = [...x.toString()]
    let xStringReversed = []

    for (let i = 0; i < xString.length; i++) {
        xStringReversed.push(xString[xString.length - i - 1]);
    }

    if (xString.toString() === xStringReversed.toString()) return true
    else return false
};